Scripts/project.swift

```diff
@@ -43,10 +43,23 @@ public func setup(name: String, orga: String) throws {
 
 /// Looks up the correct namespaces and adds them in IB files. Use only after namespacing all Image Asset folders.
 public func namespaceImages() throws {
-    let assetPaths = run(bash: "LC_ALL=C find App/Resources/Images.xcassets/ -type d -name *.imageset").stdout.components(separatedBy: .newlines)
+    try namespaceAssetCatalog(withTitle: "Images")
+}
+
+/// Looks up the correct namespaces and adds them in IB files. Use only after namespacing all Color Asset folders.
+public func namespaceColors() throws {
+    try namespaceAssetCatalog(withTitle: "Colors")
+}
+
+/// Looks up the correct namespaces and adds them in IB files. Use only after namespacing all Asset folders.
+private func namespaceAssetCatalog(withTitle title: String) throws {
+    let assetPaths = run(bash: "LC_ALL=C find App/Resources/\(title).xcassets/ -type d \\( -name \"*.colorset\" -or -name \"*.imageset\" \\)").stdout.components(separatedBy: .newlines)
```

Comment:
Can you use raw string (`#"string"#`) here to minimize the escaping. Also, this code is only correct if the `Colors.xcasset` doesn't include any images and vice versa. Just to be sure, I think, we should change that. Having that said, it may make more sense to use an enum with the cases `colors` and `images` as parameter and set the string values needed appropriately.